File: bench/cases/array-copy-1m/main.ts
let src: number[] = []
for (let i = 0; i < 10000000; i++) {
  src.push(i)
}
let dst: number[] = []
for (let i = 0; i < src.length; i++) {
  dst.push(src[i])
}
console.log(dst.length + dst[9999999])

File: bench/cases/array-copy-1m/main.go
package main

import "fmt"

func main() {
	src := make([]int64, 0)
	for i := int64(0); i < 10000000; i++ {
		src = append(src, i)
	}
	dst := make([]int64, 0)
	for i := 0; i < len(src); i++ {
		dst = append(dst, src[i])
	}
	fmt.Println(int64(len(dst)) + dst[9999999])
}
